### PR TITLE
chore: update ext/http to hyper 1.0.1 and http 1.0

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/21583.

Sadly it appears we're not able to update parts of `ext/http` to Hyper 1.0.

### CODE REVIEW

File: ext/http/Cargo.toml [hunk 1]
```diff
@@ -32,9 +32,11 @@ deno_net.workspace = true
 deno_websocket.workspace = true
 flate2.workspace = true
 http.workspace = true
+http_1 = { package = "http", version = "=1.0.0" }
 httparse.workspace = true
 hyper = { workspace = true, features = ["server", "stream", "http1", "http2", "runtime"] }
-hyper1 = { package = "hyper", features = ["full"], version = "=1.0.0-rc.4" }
+hyper-util = { version = "=0.1", features = ["tokio"] }
```
Comment (mmastrac) [inline ext/http/Cargo.toml:38]: Glad this was finally stabilized

**Review by mmastrac — APPROVED** [review]
LGTM. It might be better to keep http_1 as http so we can eventually just delete http_02, but I don't think that's a blocker for this to land.

File: cli/util/logger.rs [hunk 2]
```diff
@@ -38,6 +38,9 @@ pub fn init(maybe_level: Option<log::Level>) {
   )
   // https://github.com/denoland/deno/issues/6641
   .filter_module("rustyline", log::LevelFilter::Off)
+  .filter_module("swc_ecma_codegen", log::LevelFilter::Off)
+  .filter_module("swc_ecma_transforms_optimization", log::LevelFilter::Off)
+  .filter_module("tracing", log::LevelFilter::Off)
```
Comment (bartlomieju) [inline cli/util/logger.rs:43]: @mmastrac could you give me a hand here? Updating hyper and related crates caused some tracing/log outputs to be displayed and I had to resort to doing this filtering, but it seems brittle (I might have missed something, even though the tests pass). I'm having trouble which crate caused this and it feels like a bug in one of these crates.

**Comment by mmastrac** [comment]
I hope this small regression disappears when the tracing dep disappears but it's not clear where it comes from.

**Pull request merged**